usage/app/lib/KinesisStreamReader.scala

```diff
@@ -0,0 +1,57 @@
+package lib
+
+import java.net.InetAddress
+import java.util.UUID
+
+import com.amazonaws.auth._
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.internal.StaticCredentialsProvider
+import com.amazonaws.services.kinesis.clientlibrary.interfaces.{IRecordProcessor, IRecordProcessorFactory}
+import com.amazonaws.services.kinesis.clientlibrary.lib.worker.{InitialPositionInStream, Worker, KinesisClientLibConfiguration}
+import com.gu.logback.appender.kinesis.helpers.CustomCredentialsProviderChain
+
+class KinesisStreamReader {
```

Comment:
As this is not a general Kinesis stream reader impl, might be better to give it a more specific name (like `CrierStreamReader`).